src/GrpcProtos/capture.proto

```diff
@@ -200,6 +200,7 @@ message ApiScopeStopAsync {
   int32 pid = 1;
   int32 tid = 2;
   uint64 timestamp_ns = 3;
+  uint64 id = 4;
```

Comment:
This was simply forgotten earlier, right?

Reply:
exactly.